src/test/java/core/basesyntax/serviceimpl/DataConverterImplTest.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax.serviceimpl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class DataConverterImplTest {
+    private static final List<String> FILE_DATA = List.of("b,banana,20",
+            "b,apple,100",
+            "s,banana,100",
+            "p,banana,13",
+            "r,apple,10",
+            "p,apple,20",
+            "p,banana,5",
+            "s,banana,50");
+    private static final FruitTransaction FIRST_TRANSACTION = new FruitTransaction(
+            FruitTransaction.Operation.BALANCE,
+            "banana",
+            20);
+    private static final int FIRST_TRANSACTION_INDEX = 0;
+    private static final List<String> EMPTY_INPUT = List.of();
+    private static DataConverterImpl dataConverter;
+
+    @BeforeAll
+    static void beforeAll() {
+        dataConverter = new DataConverterImpl();
+    }
+
+    @Test
+    void convert_validInput_Ok() {
```

Comment:
```suggestion
    void convert_validInput_ok() {
```
let's follow our naming convention. This part of a method should start with non-capital letter. Fix in all places